lib/ansible/modules/cloud/azure/azure_rm_virtualmachineextension.py

```diff
@@ -102,13 +99,18 @@
 
 RETURN = '''
 state:
-    description: Current state of the vm extension
+    description:
+        - Current state of the vm extension.
     returned: always
     type: dict
+    sample: { "status":"Deleted" }
```

Comment:
Shouldn't this be `state`, not `status`?
```suggestion
    sample: { "state":"Deleted" }
```